app/isomorphic/components/layouts/footer/index.js

```diff
@@ -14,21 +14,28 @@ const generateItemsList = (item, id) => (
 );
 
 const generateMenuGroup = placeholderMenus => {
-  return placeholderMenus.map(({ title, children }, id) => (
-    <div styleName="menu-group" key={id}>
-      <div styleName="footer-headings">{title}</div>
-      <ul>{children.map(generateItemsList)}</ul>
-    </div>
-  ));
+  return placeholderMenus.map(
+    ({ title, children }, id) =>
+      children.length > 0 && (
+        <div styleName="menu-group" key={id}>
+          <h6 styleName="title">{title}</h6>
+          <ul>{children.map(generateItemsList)}</ul>
+        </div>
+      )
+  );
 };
 
 const FooterBase = footer => {
   const placeholderMenus = footer.menu.filter(item => (item["item-type"] = "placeholder"));
 
   return (
-    <div styleName="footer">
-      <AppLogo />
-      {generateMenuGroup(placeholderMenus)}
+    <div className="container">
+      <div styleName="footer">
+        <div>
+          <AppLogo />
+        </div>
+        {generateMenuGroup(placeholderMenus)}
```

Comment:
Why are we separating `generateMenuGroup` code? can add it here itself

Reply:
@ags1773 it would be good if we keep it in a separate func. (readability)

Reply:
I think it should be a seperate component. Dont make it a function.